src/usr/local/www/diag_packet_capture.php

```diff
@@ -479,7 +482,14 @@ function fixup_host($value, $position) {
 	}
 
 	if ($port != "") {
-		$matches[] = "port ".fixup_not($port);
+		for($i = 0; $i < count($ports); $i++){
+                    	 if($i == count($ports)-1) {
+                	                $ports_matches .= "port ".($ports[$i]);
```

Comment:
This lost the `fixup_not()` used in the previous code, which will make negating ports impossible. Perhaps that's why there are parenthesis where they appear to be unnecessary?

Also needs spaces around the `.`.